cluster/gce/config-default.sh

```diff
@@ -167,7 +167,7 @@ export ENABLE_DOCKER_REGISTRY_CACHE=true
 
 # Optional: Deploy a L7 loadbalancer controller to fulfill Ingress requests:
 #   glbc           - CE L7 Load Balancer Controller
-export ENABLE_L7_LOADBALANCING="${KUBE_ENABLE_L7_LOADBALANCING:-glbc}"
```

Comment:
Bad merge? Why is this getting deleted? Or is the comment above meant to be preserved?